bindings/rust/integration/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(all(feature = "network-tests", test))]
+mod network;
+
+#[cfg(test)]
+mod tests {
+    use s2n_tls::{
+        security::Policy,
+        testing::{self, TestPair},
+    };
+
+    /// This test provides a helpful debug message if the PQ feature is incorrectly
+    /// configured.
+    #[cfg(feature = "pq")]
+    #[test]
+    fn pq_sanity_check() -> Result<(), Box<dyn std::error::Error>> {
+        let config = testing::build_config(&Policy::from_version("KMS-PQ-TLS-1-0-2020-07")?)?;
```

Comment:
"[default_pq](https://github.com/aws/s2n-tls/blob/main/docs/usage-guide/topics/ch15-post-quantum.md#chart-security-policy-version-to-pq-hybrid-key-exchange-methods)" might be a better / more readable option.

Might also be worth opening a little follow-up issue to add "default_pq" to the pq integration tests :thinking:

Reply:
`default_pq` will requires us to add the `kem_group` API. 

I think it's gonna be neater to to that in a separate PR, so I opened an issue to track this: #4896 . The issue explicitly calls out
- updating the integration test
- updating the pq sanity check
as requirements for closing the issue.